tests/automation.js

```diff
@@ -123,23 +116,18 @@ casper.test.begin("unit tests", 16 + gfxTests.length, function(test) {
     function basicUnitTests() {
         casper.waitForText("DONE", function() {
             var content = this.getPageContent();
-            var regex = /DONE: (\d+) pass, (\d+) fail, (\d+) known fail, (\d+) unknown pass/;
+            var regex = /DONE: (\d+) class pass, (\d+) class fail/;
             var match = content.match(regex);
-            if (!match || !match.length || match.length < 5) {
+            if (!match || !match.length || match.length < 3) {
                 this.echo("data:image/png;base64," + this.captureBase64('png'));
                 test.fail('failed to parse status line of main unit tests');
             } else {
-                var msg = "";
-                for (var i = 0; i < expectedUnitTestResults.length; i++) {
-                    if (match[i+1] != expectedUnitTestResults[i].number) {
-                        msg += "\n\tExpected " + expectedUnitTestResults[i].number + " " + expectedUnitTestResults[i].name + ". Got " + match[i+1];
-                    }
-                }
-                if (!msg) {
+                var failed = match[2];
+                if (failed === "0") {
                     test.pass('main unit tests');
                 } else {
                     this.echo("data:image/png;base64," + this.captureBase64('png'));
```

Comment:
@mykmelez  It seems `this.echo("data:image/png;base64," + this.captureBase64('png'));` is redundant? Everytime the unit tests fails, it logs a bunch of annoying base64 string on travis.